src/components/registration/steps.jsx

```diff
@@ -445,6 +436,13 @@ class DemographicsStep extends React.Component {
         this.setState({otherDisabled: gender !== 'other'});
     }
     handleValidSubmit (formData) {
+        // look up country name using user's country code selection
+        if (formData.countryCode) {
```

Comment:
Similarly, we should validate that there's a `countryCode` before this point, and if it's not supplied, enter into an error state.

Reply:
OK, now it should pop up a validation error before it gets to this point, and if it doesn't find a country name, it won't call `onNextStep`.